staging/src/k8s.io/kubectl/pkg/cmd/drain/drain.go

```diff
@@ -343,7 +343,7 @@ func (o *DrainCmdOptions) deleteOrEvictPodsSimple(nodeInfo *resource.Info) error
 	}
 
 	if err := o.drainer.DeleteOrEvictPods(list.Pods()); err != nil {
-		pendingList, newErrs := o.drainer.GetPodsForDeletion(nodeInfo.Name)
+		pendingList, newErrs := o.drainer.GetPodsForDeletion(nodeInfo.Name, list.ResourceVersion)
```

Comment:
This seems like it would just fetch the cached list of pods from the previous list request?  Under what circumstances will a new list make it into the api server cache?  That's unclear from the documentation.

If the list is being periodically generated by the API server cache, it seems unlikely that a single additional call is going to cause any impact.  If it's not, then we'll either have stale data here or save 0 calls to the storage backend.